Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Plan.hs

```diff
@@ -143,15 +147,26 @@ currentPlan (driverId, _merchantId) = do
   mPlan <- maybe (pure Nothing) (\p -> QPD.findByIdAndPaymentMode p.planId (getDriverPaymentMode driverInfo.autoPayStatus)) mDriverPlan
   mandateDetailsEntity <- mkMandateDetailEntity (join (mDriverPlan <&> (.mandateId)))
   currentPlanEntity <- maybe (pure Nothing) (convertPlanToPlanEntity driverId >=> (pure . Just)) mPlan
-
+  latestManualPayment <- QDF.findLatestByFeeTypeAndStatus DF.RECURRING_INVOICE DF.CLEARED driverId
```

Comment:
Lets pass status as array `[CLEARED, COLLECTED_CASH]` to support cash collection feature in future too.